### DISCUSSION TITLE
V2X simulation in CARLA

### DISCUSSION THREAD

**Discussion by PooriyaTT** [carla-simulator/carla#5962] [General]
Hello guys, 
I am trying to simulate V2X communication (to be more specific V2I communication) with a single RSU and OBU to communicate necessary information between infrastructure and the ego vehicle. So, my question is, is it possible to simulate V2I communication with CARLA? Or is there any tutorial for it?

**Comment by CHILLed144** [comment]
hey, i know it's been a long time, but did you manage to do it?
